business/definitionService.js

```diff
@@ -261,9 +258,9 @@ class DefinitionService {
 
   // Compute and store the scored for the given definition but do it in a way that does not affect the
   // definition so that further curations can be done.
-  async _ensureToolScores(coordinates, definition) {
+  async _ensureToolScores(coordinates, definition, raw) {
```

Comment:
this whole call chain no longer needs async

Reply:
Actually this method could be removed at all. I will fix it. Thanks!

Reply:
not sure what you mean about removing methods. All the methods here were here before and need to stay. Just the async/awaits no longer need to be added.

Reply:
Ah, sorry, i thought you were referring to the `ensureUrls` method that actually is not used anymore. My bad.